.travis.yml

```diff
@@ -40,7 +42,9 @@ before_script:
   - composer update --prefer-source --no-interaction $COMPOSER_FLAGS
   - sudo pip install -r Resources/doc/requirements.txt
 
-script: make test
+script:
+ - make test
+ - if [ "$CS_FIXER" = "run" ]; then make cs_dry_run ; fi;
```

Comment:
If it's on `before_script`, Travis would throw an error.

If it's on `script`, Travis would throw a failed test.

Would be better to be on `script` so but yes, before phpunit.